packages/ng-primitives/dialog/src/dialog-trigger/dialog-trigger.directive.ts

```diff
@@ -24,6 +25,7 @@ import { NgpDialogTriggerToken } from './dialog-trigger.token';
   standalone: true,
   selector: '[ngpDialogTrigger]',
   exportAs: 'ngpDialogTrigger',
+  hostDirectives: [NgpFocusVisible],
```

Comment:
Unfortunately I don't believe we can have this here, for example, if the trigger is an `ngpButton` then `NgpFocusVisible` would be applied by both `ngpButton` and `ngpDialogTrigger` which would throw an issue.
I guess the solution would be to use `ngpButton` on the trigger in the example.

Reply:
You're right. I'll send an update. So, the same problem is with ngpAccordionTrigger AFIAK.

Reply:
Well spotted, that is a good point, the same change should probably be made for it! Thank you!

Reply:
fixed.